inline/src/lexer/mod.rs

```diff
@@ -558,7 +569,9 @@ impl Iterator for Tokens {
             self.iter.next()?
         };
 
-        match unr_token.pop() {
+        let part = unr_token.pop();
+
+        match part {
```

Comment:
why this change?
maybe leftover from debugging?

Reply:
Correct!